tidb-cloud/tidb-cloud-upload-local-files.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Upload Local Files to TiDB Cloud
+summary: Learn how to upload local files to TiDB Cloud.
+---
+
+# Upload Local Files to TiDB Cloud
+
+You can upload local files to TiDB Cloud directly. You only need to select a local file, select the target cluster and table, make changes if needed, and then directly import the data into TiDB Cloud.
+
+This feature now supports the following:
+
+- Uploading one CSV file within 50 MiB for one task
+- Only Serverless Tier clusters
+
+## Prerequisites
+
+- Prepare the local file to be uploaded. The file must be in CSV format within 50 MiB.
+- Use [SQL Editor](/develop/dev-guide-tidb-crud-sql.md#explore-sql-with-tidb) to create a table in the target database. The table schema must be consistent with the local file. Alternatively, you can create a table when filling in the import task.
```

Comment:
Use [SQL Editor](/develop/dev-guide-tidb-crud-sql.md#explore-sql-with-tidb) to create a table in the target database in advance.The table schema must be consistent with the local file. Alternatively, You can enter the table name in the Import task and let the program help you automatically create the target table.